bin/index.js

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env node
+
+import yargs from 'yargs';
+import { hideBin } from 'yargs/helpers';
+import chalk from 'chalk';
+import readline from 'readline';
+
+import config from '../src/lib/config.js';
```

Comment:
for me looks like bin should not depend on the src.

index should be in the src. bin should be prepared somehow (build, copied etc) but should not import anything from source.